src/main/config/index.ts

```diff
@@ -20,3 +23,23 @@ export const LIMITED_NAME_REGISTERED_WITH_AML = `${BASE_LIMITED_TRADER_URL}/name
 export const LIMITED_BUSINESS_MUSTBE_AML_REGISTERED = `${BASE_LIMITED_TRADER_URL}/business-mustbe-aml-registered/business-mustbe-aml-registered`;
 export const UNINCORPORATED_WHAT_IS_THE_BUSINESS_NAME = `${BASE_UNINCORPORATED_URL}/what-is-the-business-name/what-is-the-business-name`;
 export const LIMITED_COMPANY_INACTIVE = `${BASE_LIMITED_TRADER_URL}/company-inactive/company-inactive`;
+export const ACSP_SERVICE_HOST = "localhost";
+export const ACSP_SERVICE_PORT = "18644";
+export const ACSP_SERVICE_BASE = `${ACSP_SERVICE_HOST}:${ACSP_SERVICE_PORT}/acsp-api`;
+export const ACSP_SERVICE_TRANSACTION_URI = `${ACSP_SERVICE_BASE}/transactions`;
+export const API_URL = getEnvironmentValue("API_URL", "http://api.chs.local:4001");
```

Comment:
Can we keep environment values and controller paths in separate file? There is file in utils properties.ts where all environment values can go